pkgs/tools/networking/v2ray/generic.nix

```diff
@@ -13,7 +13,7 @@ let
     pname = "v2ray-core";
     inherit version src;
 
-    modSha256 = "11gsncy3449a7y6w6pr7acqabyj2q2a1q52f8fcl5cdz1vjbmmxi";
```

Comment:
this one is in the wrong commit

Reply:
Moved - that was the one I did by hand to test :)